### DISCUSSION TITLE
Repositories for an organisation

### DISCUSSION THREAD

**Discussion by spbarnes** [octokit/octokit.net#2707] [Q&A]
How can you get a list of repositories for a specific organization?

**Comment by kfcampbell** [comment]
Can you use the RepositoriesClient's [GetAllForOrg](https://github.com/octokit/octokit.net/blob/main/Octokit/Clients/RepositoriesClient.cs#L462) method to do so?

**Comment by drewid** [comment]
`var repositories = await client.Repository.GetAllForOrg("OrgName")`
